layers/cluster/kvm/cluster-kvm-openChami.yaml

```diff
@@ -185,24 +243,25 @@ cluster:
         # This needs to be coordinated with the application metadata in
         # the provider layer config to make sure Blade xnames match up
         # with node xnames. Blade xnames are currently defined there up
-        # to 16 blades (support for 64 nodes)
+        # to 16 blades (support for 64 nodes). Currently defined 16
+        # compute nodes (four per blade 'n0' through 'n3').
         node_names:
```

Comment:
REVIEWERS: this is a bit gratuitous, but I changed all the node xnames over to live on the same shelf just different blades (BMCs). We have 4 compute nodes per BMC and 4 BMCs configured here for a total of 16 nodes. There is a pretty good chance that when I implement VSHA-700 I will remove this configuration and have the application layer compute the node xnames based on the blade xnames and the node instance within the blade, and then register the resulting node names with the cluster layer, but, for now, this is what we have.